configuration/driver/icedrv_InitMod.F90

```diff
@@ -97,13 +99,18 @@ subroutine icedrv_initialize
       endif
 
       if (tr_fsd) then
+         do i=1,nx
```

Comment:
What changes with grid cell in icepack_init_fsd_bounds, that would cause the fsd size arrays to be different?